projects/libyuv/libyuv_fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
```

Comment:
since AFL and Honggfuzz don't respect max_len we generally discourage doing this.
It's better to enforce this in LLVMFuzzerTestOneInput.

Reply:
ok, got it.

Reply:
In the line
   libyuv::I420ToNV21(src_y_data, width, src_u_data, (width + 1) / 2, src_v_data, (width + 1) / 2, dst_y_data, width, dst_uv_data, width, width, height);

The 3rd to last value is a stride that may be affected by odd width subsampling. 
It should be( (width + 1) / 2) * 2 or (width + 1) & ~1

Reply:
Thank you for your reply! This is my mistake, so I have modified this line of code and show blow.
**libyuv::I420ToNV21(src_y_data, width, src_u_data, (width + 1) / 2, src_v_data, (width + 1) / 2, dst_y_data, width, dst_uv_data,(width+1)/2*2, width, height);**